www/css/framework7.ios.colors.min.css

```diff
@@ -12,4 +12,4 @@
  * 
  * Released on: April 10, 2017
  */
```

Comment:
Yes, think I changed a color because it was marked important and I couldn't change with regular css